config/areas/languages/dialogs.php

```diff
@@ -217,18 +217,18 @@
 			return [
 				'component' => 'k-remove-dialog',
 				'props' => [
-					'text' => 'Do you really want to delete the variable for "' . $translationKey . '"?'
+					'text' => 'Do you really want to delete the variable for "' . Escape::html($variable->key()) . '"?'
```

Comment:
Do you want to turn these into i18n strings as well if you’re already touching this code?

Reply:
Oh sorry, sometimes I think too complex 😅 I'll update non-translated strings 👍